README.rst

```diff
@@ -14,14 +14,14 @@
 
 |
 
-Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a toolkit of libraries (Ray AIR) for simplifying ML compute:
+Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a set of AI libraries for simplifying ML compute:
 
 .. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/what-is-ray-padded.svg
 
 ..
   https://docs.google.com/drawings/d/1Pl8aCYOsZCo61cmp57c7Sja6HhIygGCvSZLi_AuBuqo/edit
 
-Learn more about `Ray AIR`_ and its libraries:
+Learn more about `Ray AI Libraries`_:
```

Comment:
Is AIR still a term we should use when talking to users?

I thought we just remove AIR as a concrete package but still keep the AIR term to represent the collection of AI libraries?

Reply:
I think we should refer to them as AI libraries generally (@richardliaw I really like this proposal, don't know why we went with AIR originally--- probably too catchy sounding).

Anecdotally, most users are thinking library centric anyways, so the term "AIR" rarely comes up organically.